modular_skyrat/modules/reagent_forging/code/crafting_bench.dm

```diff
@@ -0,0 +1,242 @@
+/obj/structure/reagent_crafting_bench
+	name = "forging work bench"
+	desc = "A crafting bench used to create forging items."
```

Comment:
```suggestion
	name = "forging workbench"
	desc = "A crafting bench fitted with tools, securing mechanisms, and a steady surface for blacksmithing."
```
Makes it feel more organic, as if its a proper workbench and not just 'use this to create items'
Plus, yes, workbench is one word. TIL too.